node_modules/gh-users/tests/util.js

```diff
@@ -338,6 +341,63 @@ var assertGetUsersFails = module.exports.assertGetUsersFails = function(client,
 };
 
 /**
+ * Assert that the Terms and Conditions can be retrieved
+ *
+ * @param  {RestClient}         client                          The REST client to make the request with
+ * @param  {Function}           callback                        Standard callback function
+ * @param  {Object}             callback.termsAndConditions     The Terms and Conditions
+ * @throws {AssertionError}                                     Error thrown when an assertion failed
+ */
+var assertGetTermsAndConditions = module.exports.assertGetTermsAndConditions = function(client, callback) {
+    client.user.getTermsAndConditions(function(err, termsAndConditions) {
+        assert.ok(!err);
+        assert.ok(termsAndConditions);
+        assert.ok(termsAndConditions.text);
+        assert.ok(termsAndConditions.lastUpdate);
+        return callback(termsAndConditions);
+    });
+};
+
+/**
+ * Assert that a user can accept the Terms and Conditions
+ *
+ * @param  {RestClient}         client                          The REST client to make the request with
+ * @param  {Number}             id                              The id of the user for which to accept the Terms and Conditions
+ * @param  {Function}           callback                        Standard callback function
+ * @throws {AssertionError}                                     Error thrown when an assertion failed
+ */
+var assertAcceptTermsAndConditions = module.exports.assertAcceptTermsAndConditions = function(client, id, callback) {
+    var start = moment();
+    client.user.acceptTermsAndConditions(id, function(err, termsAndConditionsStatus) {
+        assert.ok(!err);
+        assert.ok(termsAndConditionsStatus);
+        assert.ok(termsAndConditionsStatus.accepted);
+        var accepted = moment(termsAndConditionsStatus.accepted);
+        assert.ok(accepted.diff(start) > 0);
```

Comment:
I think this is brittle. I've seen same-ms updates when I did oauth, unfortunately. I think for the sanity check, we can do `>=` instead of `>`, and then have a test that specifically ensures the timestamp updates with a delay of 100ms or something.